udata/core/post/factories.py

```diff
@@ -7,6 +7,15 @@
 from .models import Post
 
 
+class UnpublishedPostFactory(ModelFactory):
```

Comment:
I'm not sure you need this, can't you use PostFactory with `published=None`?

Reply:
Yes my bad